1.10/deploying-jobs/quickstart.md

```diff
@@ -72,7 +72,7 @@ You can create and manage jobs from the DC/OS CLI using `dcos job` commands. To
     }
     ```
 
-     **Note:** You can only assign one schedule to a job.
+   You can only assign one schedule to a job.
```

Comment:
I think there's a misunderstanding. I wasn't recommending that you remove the **Note:**. I think the word "only" appear should before "one schedule", not "assign".  IOW:

**Note:** You can assign only one schedule to a job.